src/codec/h264/parser.rs

```diff
@@ -916,12 +916,6 @@ impl SpsBuilder {
         self
     }
 
-    pub fn resolution_in_mbs(mut self, width: u32, height: u32) -> Self {
-        self.0.pic_width_in_mbs_minus1 = width - 1;
-        self.0.pic_height_in_map_units_minus1 = height - 1;
-        self
-    }
-
```

Comment:
Let's skip this one for now.

Reply:
Mmm or maybe not. The purpose of cros-codecs is not to be a tool to produce invalid bitstreams, it's quite the contrary - we want to use this in consumer devices, so we should make it as difficult as possible to produce an invalid stream. The type limitations that this series applies to the SPS already make it impossible to output some of these streams anyway (i.e. can't output a 32-bit width), so even if we leave the option open it cannot be used as a complete research/attach tool.

So on second thought, I am rather of the opinion that we should enforce consistency in the API whenever possible, and at runtime otherwise.